DataType/PercentDataType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\DataType;
+
+use Netdudes\DataSourceryBundle\Query\FilterCondition;
+
+class PercentDataType extends AbstractDataType
+{
+    /**
+     * Return the identifier of the default sort method.
+     *
+     * @return int
+     */
+    public function getDefaultFilterMethod()
+    {
+        return FilterCondition::METHOD_NUMERIC_EQ;
+    }
+
+    /**
+     * Return the name of the type, unique.
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return 'percentage';
```

Comment:
Is it the same as in `DataSourceBuilder::getDataTypeByName()`?
Shouldn't we use some constants for this?
But probably out of scope.